### PR TITLE
nymvpncli guide 

### PR DESCRIPTION
**Pull request by mfahampshire**
- [x] modify to use `nym-vpnc` instead 

This change is [Reviewable](https://reviewable.io/reviews/nymtech/nym/5243)

### CODE REVIEW

**Review by tommyv1987 — APPROVED** [review]

File: documentation/docs/pages/developers/nymvpncli.mdx [hunk 1]
```diff
@@ -0,0 +1,278 @@
+import { Callout } from 'nextra/components'
+
+# Nym VPN CLI
+
+This is a short guide to setting up and using the `nym-vpnc` tool, which is used in conjunction with the `nym-vpnd` daemon.
+
+<Callout type="warning" emoji="⚠️">
+  These binaries have superceded the older `nym-vpn-cli` binary. This still operates for the moment as it is being used in testing scenarios but will go out of date quickly.
```
Comment (serinko) [inline documentation/docs/pages/developers/nymvpncli.mdx:8]: Not sure if the white line in the beginning gets propagated.

File: documentation/docs/pages/developers/nymvpncli.mdx [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+</Callout>
+
+Download and run instructions for the GUIs can be found [here](https://nymvpn.com/en/download/linux).
+
+## Download & Extract Binary
+Check the [release page](https://github.com/nymtech/nym-vpn-client/releases/) page for the latest release version and modify the instructions accordingly. These instructions use the latest as of the time of writing.
+```sh
+wget -q https://github.com/nymtech/nym-vpn-client/releases/download/nym-vpn-core-v1.1.0-beta.3/nym-vpn-core-v1.1.0-beta.3_<YOUR_OPERATING_SYSTEM>.tar.gz &&
```
Comment (serinko) [inline documentation/docs/pages/developers/nymvpncli.mdx:16]: Isn't here a way to put latest tag or release tag to not to have to worry about updating it?

File: documentation/docs/pages/developers/nymvpncli.mdx [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+tar -xzf nym-vpn-core-v1.1.0-beta.3_<YOUR_OPERATING_SYSTEM>.tar.gz &&
```
Comment (serinko) [inline documentation/docs/pages/developers/nymvpncli.mdx:17]: or alternatively make it a bit more eye banging that a user has to think and replace:

`tar -xzf <BINARY>.tar.gz`

File: documentation/docs/pages/developers/nymvpncli.mdx [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+yay -S gcc make protobuf base-devel clang
+```
+
+**Ubuntu24 specific packages:**
+```sh
+apt install gcc make protobuf-compiler pkconfig libdbus-1-dev build-essential clang
+```
+
+<Callout type="warning" emoji="⚠️">
+  Older Debian/Ubuntu versions need to manually install `protobuf-compiler` >= v3.21.12
```
Comment (serinko) [inline documentation/docs/pages/developers/nymvpncli.mdx:37]: Also starts with a whitespace

File: documentation/docs/pages/developers/nymvpncli.mdx [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+cd nym-vpn-client/
+make
+```
+
+## Mnemonic Generation
+Create an account at [nymvpn.com](nymvpn.com) to obtain your mnemonic.
+
+## Start the daemon
+```sh
+sudo ./PATH/TO/nym-vpnd
```
Comment (serinko) [inline documentation/docs/pages/developers/nymvpncli.mdx:52]: I would consider to:
```
import { VarInfo } from 'components/variable-info.tsx';
```

Then put 
```
<VarInfo />
```
Somewhere on top and use the PATH cnvention like it's noted there. 

https://nymtech.net/docs/operators/variables

Which is `<PATH_TO>`

File: documentation/docs/pages/developers/nymvpncli.mdx [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+2024-12-11T11:03:59.982230Z  INFO nym_vpn_account_controller::shared_state: Setting mnemonic state to NotStored
+2024-12-11T11:03:59.982240Z  WARN nym_vpn_account_controller::commands: Returning error: NoAccountStored
+```
+
+Ignore the `NoAccountStored` errors: these will disappear after the next step. **Leave the daemon running and run the following commands in another terminal window** or create an init file for `nym-vpnd`.
+
+## Run VPN
+We have to first store the account we have created online:
+```sh
+./PATH/TO/nym-vpn-cli store-account --mnemonic "<MNEMONIC_FROM_GENERATION_STEP>"
```
Comment (serinko) [inline documentation/docs/pages/developers/nymvpncli.mdx:84]: I would consider to:
```
import { VarInfo } from 'components/variable-info.tsx';
```

Then put 
```
<VarInfo />
```
Somewhere on top and use the PATH cnvention like it's noted there. 

https://nymtech.net/docs/operators/variables

Which is `<PATH_TO>`

File: documentation/docs/pages/developers/nymvpncli.mdx [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+    last_updated_utc: "2024-12-11 11:04:36.432Z",
+    device_identity_key: "BZWA5MRnEvRYD8WWrH9KULdj2Q1uTssu6idjgWFae9dv",
+    status: Active,
+} id=09876a3a
+2024-12-11T11:04:36.765998Z  INFO register_device: nym_vpn_account_controller::commands::register_device: Device registered: BZWA5MRnEvRYD8WWrH9KULdj2Q1uTssu6idjgWFae9dv id=09876a3a
+```
+
+You can then connect `nym-vpnc` (in this case, with 2 hop wireguard mode enabled):
+```sh
+./PATH/TO/nym-vpn-cli nym-vpnc connect --enable-two-hop
```
Comment (serinko) [inline documentation/docs/pages/developers/nymvpncli.mdx:111]: I would consider to:
```
import { VarInfo } from 'components/variable-info.tsx';
```

Then put 
```
<VarInfo />
```
Somewhere on top and use the PATH cnvention like it's noted there. 

https://nymtech.net/docs/operators/variables

Which is `<PATH_TO>`

**Review by serinko — APPROVED** [review]
Levft a few comments but it's not blocking, decide how to handle them - APPROVED

File: documentation/docs/pages/developers/nymvpncli.mdx (hunk 2 above)
Comment (serinko) [reply documentation/docs/pages/developers/nymvpncli.mdx:16]: Or use `<BINARY>` so ppl copy-paste the whole thing anyway and you have less future management work

**Pull request merged**